### PR TITLE
Allow steambot to get a translated schema from steam

### PR DESCRIPTION
**Pull request by WildCard65**
Defaults to system's current UI lang if incorrect value is set or
missing in config.

### CODE REVIEW

File: Bin/Debug/settings-template.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
             "DisplayNamePrefix":"[AutomatedBot] ",
             "TradePollingInterval":800,
             "LogLevel":"Success",
-            "AutoStart": "true"
+            "AutoStart": "true",
+            "SchemaLang":"2 letter code for language schema should be translated in"
```
Comment (scholtzm) [inline Bin/Debug/settings-template.json]: I would make the default value English ("en").

Comment (WildCard65) [inline Bin/Debug/settings-template.json]: In Bot.cs I set the default to null.

**Comment by scholtzm** [comment]
I would probably make the default value everywhere English. Using any country code available doesn't make that much sense since the schema doesn't support every language out there.

**Comment by debugger-perm** [comment]
IMHO better way - add multilangual support for schema. for example - before
trade check user language in profile and use this language in messages.

2015-01-22 18:04 GMT+05:00 WildCard65 <notifications@github.com>:

**Comment by WildCard65** [comment]
Can we even get a user's language?

**Comment by debugger-perm** [comment]
Yep. f.e. we can check ISteamUser/GetPlayerSummaries steam web api method
(loccountrycode), ask user, analyse user messages to detect language, or
create site for users where they can select language. Different brains,
different ways ;)

2015-01-23 4:02 GMT+05:00 WildCard65 <notifications@github.com>:

**Comment by WildCard65** [comment]
Yes but there's another issue... The schema is only fetched once b4 any user can interact with the bot.

**Comment by debugger-perm** [comment]
No. The schema fetched once at bot login.

2015-01-23 5:02 GMT+05:00 WildCard65 <notifications@github.com>:

**Comment by WildCard65** [comment]
Yes and that happens b4 users can interact with the bot(aka, b4 the UserHandler gets it's OnLoginCompleted callback fired)

**Comment by scholtzm** [comment]
That makes no sense. 99% people uses english when trading and they don't expect anything else. You would also eventually have to fetch all the different schemas. Seems like a complete overkill.

**Comment by WildCard65** [comment]
Which I originally planned on making the default null if none was found, but then I remember that the CultureInfo had ISO2LetterCode property.

File: Bin/Debug/settings-template.json (hunk 1 above)
Comment (BlueRaja) [inline Bin/Debug/settings-template.json]: You should set the default here to "en" as well.  You can add a comment using `//`, though I don't think even that is necessary.  Most people will have seen the two-letter ISO abbreviation for their own language before.

**Pull request merged**